AppCenter/AppCenter/Internals/Context/UserId/MSUserIdContextDelegate.h

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+NS_ASSUME_NONNULL_BEGIN
+
+@class MSUserIdContext;
+
+@protocol MSUserIdContextDelegate <NSObject>
+
+@optional
+
```

Comment:
Why optional? We have only one method here :) Want it - implement the protocol, don't want it - don't implement.

Reply:
Reasonable, ill fix this. thx o/